components/Table/Table.js

```diff
@@ -3,61 +3,95 @@ export class Table {
   tableContainer = null;
   tableBody = null;
   tableHeader = null;
+  callbackFunc = null;
+  isOrderAsc = false;
 
-  constructor(columnsConfig, tableContainer) {
+  constructor(columnsConfig, tableContainer, callbackFunc) {
     this.columnsConfig = columnsConfig;
     this.tableContainer = tableContainer;
+    this.callbackFunc = callbackFunc;
   }
 
   createTable() {
-    this.tableHeader = document.createElement("div");
-    this.tableHeader.classList.add("table-header");
+    this.tableHeader = document.createElement('div');
+    this.tableHeader.classList.add('table-header');
     this.tableContainer.append(this.tableHeader);
 
-    this.tableBody = document.createElement("div");
-    this.tableBody.classList.add("table-body");
+    this.tableBody = document.createElement('div');
+    this.tableBody.classList.add('table-body');
     this.tableContainer.append(this.tableBody);
 
     this.createHeader();
   }
 
   createHeader() {
-    const tableHeaderRow = document.createElement("div");
-    tableHeaderRow.classList.add("table-header-row");
+    const tableHeaderRow = document.createElement('div');
+    tableHeaderRow.classList.add('table-header-row');
 
     const cells = this.columnsConfig.map((col) => {
-      const cell = document.createElement("div");
-      cell.classList.add("table-header__cell");
+      const cell = document.createElement('div');
+      cell.classList.add('table-header__cell');
+      cell.setAttribute('data-key', col.key);
+      cell.setAttribute('data-sortable', col.sortable);
+      const headerLabel = document.createElement('div');
+      headerLabel.classList.add('table-header-label');
 
-      const actionsContainer = document.createElement("div");
-      actionsContainer.classList.add("actions-container");
+      if (col.sortable) {
+        headerLabel.addEventListener('click', () => {
+          this.isOrderAsc = !this.isOrderAsc;
+          const dataKey = cell.getAttribute('data-key');
+          arrowBtn.setAttribute('asc', this.isOrderAsc);
```

Comment:
You're declaring `arrowBtn` below, please do not use it before the declaration
btw, you're setting the same attribute below and we discussed regarding the naming convention